bootstrap/src/main/java/com/facebook/airlift/bootstrap/Bootstrap.java

```diff
@@ -291,4 +292,13 @@ private static ColumnPrinter makePrinterForConfiguration(ConfigurationFactory co
         }
         return columnPrinter;
     }
+
+    private boolean getDefaultFromProperties(String propertyName, boolean defaultValue)
+    {
+        Map<String, String> properties = getSystemProperties();
```

Comment:
wondering if we can use `System.getProperty` instead of loading all the properties.